integration_test/common/common.go

```diff
@@ -81,19 +81,19 @@ type ConfigurationOptions struct {
 
 type IntegrationMetadata struct {
 	PublicUrl                    string                       `yaml:"public_url"`
-	AppUrl                       string                       `yaml:"app_url" validate:"required"`
-	ShortName                    string                       `yaml:"short_name" validate:"required"`
-	LongName                     string                       `yaml:"long_name" validate:"required"`
+	AppUrl                       string                       `yaml:"app_url" validate:"required,url"`
+	ShortName                    string                       `yaml:"short_name" validate:"required,excludesall=‘’“”"`
+	LongName                     string                       `yaml:"long_name" validate:"required,excludesall=‘’“”"`
 	LogoPath                     string                       `yaml:"logo_path"`
-	Description                  string                       `yaml:"description" validate:"required"`
+	Description                  string                       `yaml:"description" validate:"required,excludesall=‘’“”"`
```

Comment:
There are also LogField's description and ConfigurationFields's description. Shall we add this `excludesall=‘’“”` to those too?

Reply:
Yeah, I meant to do that but forgot. Thanks for the reminder. Done.